libs/remix-url-resolver/src/github-folder-resolver.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line no-unused-vars
+import axios, { AxiosResponse } from 'axios'
+
+export const githubFolderResolver = async (url, obj = {}, maxDepth, depth?, rootPath?) => {
+  depth = depth ? depth : 0
+  const child = await pullFolder(url)
+  depth = depth++
+  const urlObj = new URL(url);
+  const pathname = urlObj.pathname;
+  const pathParts = pathname.split('/');
+  const folderPath = pathParts.slice(5).join('/');
+  rootPath = rootPath || folderPath
+  for (const item of child) {
```

Comment:
Add a check to ensure that `child` can be iterated over, because it throws an error when i pass in a path that points to a github file.